articles/030_ros_with_zeromq.md

```diff
@@ -18,7 +18,7 @@ author: '[William Woodall](https://github.com/wjwwood)'
 {{ page.abstract }}
 </div>
 
-> **This document pre-dates the decision to build ROS 2 on top of DDS.**
```

Comment:
The linter warning could be read as: "avoid standalone fully emphasized lines, because they may become rendered too look like section headings. ". You could raise attention like this instead: "*Warning:* ..."